docs/text/rolling.rst

```diff
@@ -21,6 +21,11 @@ the last 10 steps.
 A rolling mechanism creates such time series for every time step by creating sub time series of the sensor data of the
 last 10 time steps.
 
+<!---
+The above scenario cannot be solved by tsfresh, can it?
```

Comment:
Yes, but you cannot do the following (correct me if I am wrong):
Extract all sub time series of length 10 from a given time series.
But that is what you usually want to do for training. Take into account the current values of the time series and the most recent (e.g. 10) values of the past.

The `maximum_number_of_timeshifts` controls how many shifts (of length 1) to perform into the past.